build.gradle.kts

```diff
@@ -153,6 +157,17 @@ allprojects {
   // (and also between JGit&co.'s slf4j-api and Intellij's slf4j-api), we need to exclude the former
   // from ALL dependencies.
   configurations.runtimeClasspath { exclude(group = "org.slf4j", module = "slf4j-api") }
+
+  tasks.withType<KotlinCompile>().configureEach {
+    val kotlinLanguageVersion = "1.7"
```

Comment:
Keep this value in configuration (maybe even intellij-versions, so as to remember to update together with minimum intellij version)